modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -894,7 +894,7 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
    * @throws {InvalidAddressDerivationPropertyError}
    * @throws {UnexpectedAddressError}
    */
-  isWalletAddress(params: VerifyAddressOptions): boolean {
```

Comment:
The `async` keyword is not actually necessary here (or in other files where we've changed the `isWalletAddress` method signatures but aren't actually using `await` within the method).